data_structures/linked_list/__init__.py

```diff
@@ -25,6 +25,30 @@ def remove(self):
     def is_empty(self):
         return self.head is None
 
+    def __str__(self):
```

Comment:
```suggestion
    def __str__(self) -> str:
```
Please add a return type hint.